### PR TITLE
Distributed autograd rST to MyST md

### LINKED ISSUE #182503: [Docathon] Convert `rpc/distributed_autograd.rst` from rST to MyST Markdown
## Description

Convert [`docs/source/rpc/distributed_autograd.rst`](https://github.com/pytorch/pytorch/blob/main/docs/source/rpc/distributed_autograd.rst) from reStructuredText to MyST Markdown and ensure that doc tests pass.

## Steps

1. **Assign the issue** to yourself using the `/assigntome` comment.

2. **Rename the file using `git mv`:**
   ```bash
   git mv docs/source/rpc/distributed_autograd.rst docs/source/rpc/distributed_autograd.md
   ```
   **Important!** Use `git mv` to convert the file — DO NOT delete the `.rst` and create a new `.md`.
   We won't accept PRs with deleted `.rst` files. We want to preserve the git log.

3. **Convert the content** from rST to MyST Markdown syntax.

4. **Build the docs locally** as described in the [README](https://github.com/pytorch/pytorch/tree/main/docs).
   Preview locally with `make serve` to ensure the HTML renders correctly.

5. **Run lintrunner** from the root of the repository:
   ```bash
   pip install lintrunner
   lintrunner init
   lintrunner -m main
   ```

6. **Submit a PR** with your changes and ensure the PR passes the doc CI tests. Specifically it needs to pass:
   - `pull / linux-docs / build-docs-python-false (pull_request)`
   - `linux-jammy-py3.9-gcc11 / test (docs_test, 1, 1, lf.linux.2xlarge)`

   Doc tests start after the Docker build, which might take about 30 minutes.
   The doc build itself should take about 25 minutes. When the build is done,
   you can see the preview of your build by clicking
   **📄 Preview Python docs built from this PR** in the pytorch-bot comment.

7. **Label your PR** with: `docathon-2026`, `module: docs`.

8. **Compare the preview** of the pages you converted with the same pages on the main branch.
   The converted pages in preview must look the same as in the main branch.

9. Once the PR is accepted, one of the maintainers will merge it.

## References

- [Quick MyST Conversion Reference](https://github.com/pytorch/pytorch/wiki/Docstring-Guidelines#restructuredtext-vs-markdown) (wiki)
- [Converter script](https://github.com/svekars/sphinx-read-thedocs-test/blob/main/rst_to_myst_md_converter.py)
- [MyST Parser documentation](https://myst-parser.readthedocs.io/en/v0.15.1/sphinx/use.html)
- [Example PR](https://github.com/pytorch/pytorch/pull/154438)

cc @sekyondaMeta @AlannaBurke

### PR DESCRIPTION
**Pull request by JonathanColetti**
fixes #182503

## Description

Converted autograd rst to myst markdown format.

Kept old version in history by running `git mv docs/source/rpc/distributed_autograd.rst docs/source/rpc/distributed_autograd.md`

Ensured linting and doc tests pass.

## Localhost docs video

https://github.com/user-attachments/assets/0107cc85-605e-43f0-bf1f-a44cf4306075

cc @svekars @sekyondaMeta @AlannaBurke

### CODE REVIEW

File: docs/source/rpc/distributed_autograd.md [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+b = torch.rand((3, 3), requires_grad=True)
+c = torch.rand((3, 3), requires_grad=True)
+d = a + b
+e = b * c
+d.sum().backward()
+```
+
+This is what the autograd graph for the code above would look like:
+
+![local dependencies](../_static/img/distributed_autograd/local_dependencies.png)
```
Comment (svekars) [inline docs/source/rpc/distributed_autograd.md:140]: To preserve the original scale, can do:

 /```{image} ../_static/img/distributed_autograd/local_dependencies.png                                       
  :scale: 80%
/```

**Comment by svekars** [comment]
LGTM, waiting for preview - one comment might want to address

**Comment by JonathanColetti** [comment]
https://docs-preview.pytorch.org/pytorch/pytorch/182717/rpc/distributed_autograd.html

What do you think? Should I adjust the scale?

cc: @svekars

**Comment by svekars** [comment]
the image looks fine, there is one issue that the date: Created On: Nov 12, 2019 change to May 06 2026. To fix that, undo your changes. Do git mv, commit, then add changes, commit, push - this should preserve git log.

**Comment by svekars** [comment]
the date seems to be still May 06 2026

**Comment by JonathanColetti** [comment]
`git log --follow -- docs/source/rpc/distributed_autograd.md` now at the end shows nov 19th 2019. Thank you for your patience @svekars
